JPS_Ontology/ontology/ontodispersion/OntoDispersion.owl

```diff
@@ -0,0 +1,477 @@
+<?xml version="1.0"?>
+<rdf:RDF xmlns="https://www.theworldavatar.com/kg/ontodispersion/"
+     xml:base="https://www.theworldavatar.com/kg/ontodispersion/"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:xml="http://www.w3.org/XML/1998/namespace"
+     xmlns:ontodispersion="https://www.theworldavatar.com/kg/ontodispersion/"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <owl:Ontology rdf:about="https://www.theworldavatar.com/kg/ontodispersion/">
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#string">17 May 2023</dc:date>
+        <owl:versionInfo rdf:datatype="http://www.w3.org/2001/XMLSchema#string">1.1</owl:versionInfo>
+        <gitCommitHash rdf:datatype="http://www.w3.org/2001/XMLSchema#string">c0599beca8df55873a1ab061dee64e52c510c6a0</gitCommitHash>
+    </owl:Ontology>
+    
+
+
+    <!-- 
+    ///////////////////////////////////////////////////////////////////////////////////////
+    //
+    // Object Properties
+    //
+    ///////////////////////////////////////////////////////////////////////////////////////
+     -->
+
+
+    
+
+
+    <!-- http://www.ontology-of-units-of-measure.org/resource/om-2/hasQuantity -->
+
+
+    <owl:ObjectProperty rdf:about="http://www.ontology-of-units-of-measure.org/resource/om-2/hasQuantity">
+        <rdfs:domain rdf:resource="https://www.theworldavatar.com/kg/ontodispersion/Emission"/>
+        <rdfs:range rdf:resource="http://www.ontology-of-units-of-measure.org/resource/om-2/Density"/>
+        <rdfs:range rdf:resource="http://www.ontology-of-units-of-measure.org/resource/om-2/MassFlow"/>
+        <rdfs:range rdf:resource="http://www.ontology-of-units-of-measure.org/resource/om-2/Temperature"/>
+    </owl:ObjectProperty>
+    
+
+
+    <!-- https://www.theworldavatar.com/kg/ontodispersion/emits -->
+
+
+    <owl:ObjectProperty rdf:about="https://www.theworldavatar.com/kg/ontodispersion/emits">
+        <rdfs:domain rdf:resource="https://www.theworldavatar.com/kg/ontodispersion/PointSource"/>
+        <rdfs:range rdf:resource="https://www.theworldavatar.com/kg/ontodispersion/Emission"/>
+        <rdfs:isDefinedBy rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://www.theworldavatar.com/kg/ontodispersion/</rdfs:isDefinedBy>
+    </owl:ObjectProperty>
+    
+
+
+    <!-- https://www.theworldavatar.com/kg/ontodispersion/emitsEmissionsAt -->
```

Comment:
This term seems a bit unnecessary, as the other relationship already states `emits`. Maybe use something along the lines of `associatedOntoCityGMLCityObject` and also add some meaningful description/label as `owl:Thing` is so generic (only necessary as OCGML has no type declarations)

The `emitsEmissionsAt` relationship (or something similar) shall be used to link the `ConsumedGasAmount` and `ProvidedHeatAmount` to the corresponding EmissionSource. Maybe a relationship stating `createsEmissionsAt` would be suitable? @kok-foong Does that make sense?
![image](https://github.com/cambridge-cares/TheWorldAvatar/assets/79832739/c0a619b3-8741-43ac-8819-a0adf1ec8a9a)
![image](https://github.com/cambridge-cares/TheWorldAvatar/assets/79832739/7b76081e-49a6-4c35-bef6-c7e1c703431b)

Reply:
Maybe there is some misunderstanding. I was thinking the ConsumedGasAmount can be linked via the isDerivedFrom relationship, happy to discuss

Reply:
This could certainly be done (and is likely the smarted way); however, that was not yet reflected in the drafted derivation cascade in Lucidchart and you didn't complain. Hence, I thought we need this relationship.
I updated Lucidchart (https://lucid.app/lucidchart/9fabc350-143b-4ca3-be52-b174c9f82c07/edit?invitationId=inv_46624da8-91ef-4def-adf7-47eec515c77e&page=0_0#) to make this relationship redundant by adding the derivation relationship as markup.

Could you please make sure that the TBox and the drafted derivation markup in Lucidchart are aligned from how you think about the problem? That's why I drafted it ;-) Thanks